### PR TITLE
Adds Edge Documentation

### PR DESCRIPTION
**Pull request by azak-azkaran**
Adds Edge Documentation
With Product Overview and Openstack introduction.
Adds small FAQ
Adds VPNaaS Documentation

### CODE REVIEW

**Review by qeqar — DISMISSED** [review]
LGTM

Tech looks good, can not say anything to the content

File: docs/_edge/faq/index.en.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: FAQ
+lang: "en"
+permalink: /edge/faq
+has_children: false
+nav_order: 8000
+---
+
```
Comment (robinnepomukmai) [inline docs/_edge/faq/index.en.md:8]: the translation is missing here @qeqar

File: docs/_edge/operationscenter/vpnaas/index.de.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: VPN as a Service
+lang: "de"
+permalink: /edge/operationscenter/vpnaas
+has_children: false
+nav_order: 2100
+parent: Operations Center
+---
+
```
Comment (robinnepomukmai) [inline docs/_edge/operationscenter/vpnaas/index.de.md:9]: German translation is missing

File: docs/_edge/productoverview/index.de.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+title: Produkt Übersicht
+lang: "de"
+permalink: /edge/productoverview
+has_children: false
+nav_order: 1000
+---
+
+# Produkt Übersicht
+
+Die ONCITE Open Edition bietet eine benutzerfreundliche virtuelle Maschine (VM) und eine Container-Plattform auf Open-Source-Basis. Sie ist skalierbar von der Größe eines Edge bis zur Größe eines Rechenzentrums und bietet volle Datenhoheit sowie Echtzeitfähigkeiten.
```
Comment (robinnepomukmai) [inline docs/_edge/productoverview/index.de.md:11]: "....von der Größe eine**R** Edge"

**Review by kbudde — APPROVED** [review]

**Review by max06 — APPROVED** [review]
Technically good. Content is up to someone else.

**Pull request merged**